commander.go

```diff
@@ -90,17 +90,18 @@ func (r *commander) Run(name string, arg ...string) {
 	r.run(false, true, name, arg...)
 }
 
-func (r *commander) run(async bool, fatal bool, name string, arg ...string) {
-	r.checkin <- struct{}{} // Check-in for work.
+
+func (r *commander) run(async bool, name string, arg ...string) {
 	cmd := exec.Command(name, arg...)
+	r.checkin <- cmd // Check-in for work.
 
 	var b bytes.Buffer
 	cmd.Stdout = &b
 	cmd.Stderr = &b
 	cmd.Dir = r.path
 	err := cmd.Start()
 	if err != nil {
-		log.Fatalf("%s\ncmd %s with err: %q at dir %q", b.String(), cmd.String(), err, r.path)
+		log.Panicf("%s\ncmd %s with err: %q at dir %q", b.String(), cmd.String(), err, r.path)
```

Comment:
I'm still on the fence whether Panicking is the best approach. I'd much prefer some other control flow.